tests/testthat/test-linear_pool.R

```diff
@@ -453,13 +583,211 @@ test_that("samples only collected and re-indexed for simplest case", {
     linear_pool(
       weights = NULL,
       task_id_cols = c("target_date", "target", "horizon", "location"),
+      compound_taskid_set = c("target", "location"),
       n_output_samples = NULL
     )
 
   expect_equal(actual_outputs, expected_outputs)
 })
 
 
+test_that("remainder samples are properly distributed when component models don't all forecast for every location", {
+  sample_outputs <- expand.grid(stringsAsFactors = FALSE,
```

Comment:
Can we add a comment here, similar to examples I added above, describing the content of the test?